.claude/commands/find-stakeholders.md

```diff
@@ -0,0 +1,111 @@
+   - Note: The API returns paginated results (100 per page). You may need to fetch multiple pages if needed.
+   - Search through the repository list to find repositories that match the components mentioned in the enhancement
+   - Look for repositories containing relevant keywords (e.g., if enhancement mentions "authentication", look for repos with "authentication" or "oauth" in the name)
+   - Common repository patterns (but don't rely solely on these):
+     - `<component>-operator` (e.g., console-operator, machine-config-operator)
+     - `cluster-<component>-operator` (e.g., cluster-authentication-operator, cluster-network-operator)
+     - Component names without "-operator" suffix (e.g., installer, hypershift, api)
+
+4. **Fetch OWNERS files from the identified repositories** to get current maintainers:
+   - Use WebFetch to retrieve OWNERS files from the matched repositories
+   - Try both `master` and `main` branches:
+     - `https://raw.githubusercontent.com/openshift/<repo-name>/master/OWNERS`
+     - `https://raw.githubusercontent.com/openshift/<repo-name>/main/OWNERS`
+   - Fetch multiple repository OWNERS files in parallel when the enhancement affects multiple components
+   - Extract the `approvers` and `reviewers` lists from each OWNERS file
+
+5. **Use local enhancement OWNERS files as supplementary context** (lower priority):
+   - Look in `enhancements/<domain>/OWNERS` files for related domains (if they exist)
+   - Available domains include: network, ingress, storage, installer, authentication, machine-config, kube-apiserver, etcd, monitoring, observability, cluster-logging, dns, security, and many others
+   - Check the root `OWNERS` file for staff engineers who handle broad-scope enhancements
+   - **Important:** Local enhancement OWNERS files may be outdated or missing. Always prioritize repository OWNERS files from step 4 over enhancement OWNERS files
+
+6. **Verify API reviewer assignment** (critical step):
+   - Check if the enhancement involves API changes (new APIs, API extensions, CRDs, webhooks, etc.)
+   - If it does, verify that the `api-approvers` field in the YAML frontmatter is populated
+   - Use WebFetch to retrieve the openshift/api OWNERS file: https://github.com/openshift/api/blob/master/OWNERS
+   - Verify that at least one person listed in the enhancement's `api-approvers` field appears in the openshift/api OWNERS file as an approver or reviewer
+   - If there is NO api-approver listed OR the listed api-approver is not in the openshift/api OWNERS file, recommend that the author reach out to the #forum-api-review Slack channel to request an API reviewer be assigned
+   - Slack channel link: https://redhat.enterprise.slack.com/archives/CE4L0F143
+
+7. **Present your findings** in this format:
+
+   ```
+   ## Potential Stakeholder Reviewers for [Enhancement Name]
+
+   ### Primary Repository: openshift/[repo-name]
+   **Approvers:**
```

Comment:
We generally want a single approver, does this help to pick a suitable single approver?